jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -231,19 +251,13 @@ private Directory attachDownloadsDirectory() {
 		return directory;
 	}
 
-	private void attachReports(RootRouter router)
-			throws ConfigurationException, ValidationException {
+	private void attachReports(RootRouter router) throws ConfigurationException, ValidationException {
 		router.attach("/reports", new ReportRouter(getContext(), router.getApplication()));
 	}
```

Comment:
attachReports() is a bit special, it does his authentication inside the ReportRouter.
This is why I didn't touch it.
I will try to see if we can use guardAndAttach() for this.